packages/sfui/frameworks/react/components/SfSelect/SfSelect.tsx

```diff
@@ -14,9 +15,38 @@ export default function SfSelect(props: SfSelectProps) {
     disabled,
     className,
     placeholder,
+    onBlur,
+    onChange,
+    onClick,
+    onKeyDown,
     ...attributes
   } = props;
 
+  const [chevronRotated, setChevronRotated] = useState(false);
+
+  const blurSelect: FocusEventHandler<HTMLSelectElement> = (value) => {
+    setChevronRotated(false);
+    return onBlur ? onBlur(value) : undefined;
+  };
+
+  const changedValue: ChangeEventHandler<HTMLSelectElement> = (value) => {
+    setChevronRotated(false);
+    return onChange ? onChange(value) : undefined;
+  };
+
+  const clickSelect: MouseEventHandler<HTMLSelectElement> = (value) => {
+    setChevronRotated(true);
+    return onClick ? onClick(value) : undefined;
+  };
+
+  const keyDownSelect: KeyboardEventHandler<HTMLSelectElement> = (value) => {
+    if (value.code === 'Space') {
+      setChevronRotated(true);
```

Comment:
What do you think about such change:
```suggestion
  const keyDownSelect = (event: KeyboardEvent<HTMLSelectElement>) => {
    if (event.key === 'space') {
      setChevronRotated(true);
```

Reply:
My bad, `event.code` is the right solution :).

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2574/commits/38b4f08ffde4f8010d16d39146832dddefe6717a